mods/ts/rules/nod-vehicles.yaml

```diff
@@ -73,6 +73,13 @@ TTNK:
 	Tooltip:
 		Name: Tick Tank
 		Description: Nod's main battle tank.\nCan deploy to gain extra protection.\n  Strong vs Vehicles\n  Weak vs Infantry, Aircraft
+		UpgradeTypes: deployed
+		UpgradeMaxEnabledLevel: 0
+	Tooltip@DEPLOYED:
+		Name: Tick Tank (deployed)
+		Description: Nod's main battle tank.\nDeployed to gain extra protection.\n  Strong vs Vehicles\n  Weak vs Infantry, Aircraft
```

Comment:
This highlights the bogosity of having this on `Tooltip` instead of `Buildable`.  These alternate descriptions will never be used.  I won't ask you to do it here (it belongs in its own PR), but we really need to move this to the proper trait.